src/main/scala/scala/collection/next/IterableOnceExtensions.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Scala (https://www.scala-lang.org)
+ *
+ * Copyright EPFL and Lightbend, Inc.
+ *
+ * Licensed under Apache License 2.0
+ * (http://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package scala.collection
+package next
+
+object IterableOnceExtensions {
+  implicit class NextIterableOnceOpsExtensions[A](private val iter: IterableOnceOps[A, Any, Any]) extends AnyVal {
```

Comment:
while this particular extension method doesn't require the `CC` or `C` parameters, future extension methods might, so I think it's worth including them for future compatibility

Reply:
You are right, my mistake, I just used **Any** to make it compile and run the tests and forgot to change it 😅